kinetic/distribution.yaml

```
@@ -2414,11 +2414,20 @@ repositories:
       version: master
     status: maintained
   earth_rover_localization:
+    doc:
+      type: git
+      url: https://github.com/earthrover/earth_rover_localization.git
+      version: 1.0.0-1
     release:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/earthrover/earth_rover_localization-release.git
       version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/earthrover/earth_rover_localization.git
+      version: 1.0.0-1
```

```suggestion
      version: master
```

Our CI requires a branch here so that it can check for updates when you make new commits. I see there's also a `develop` branch too. I don't know what your process is but am guessing master is the best one to use.